src/components/Board/BoardButtonDetails/BoardButtonDetails.css

```diff
@@ -2,14 +2,29 @@
   display: flex;
 }
 
-.BoardButtonDetails__image {
+.BoardButtonDetails__preview {
   margin-right: 16px;
+  width: 150px;
+  position: relative;
 }
 
-[dir='rtl'] .BoardButtonDetails__image {
+[dir='rtl'] .BoardButtonDetails__preview {
   margin-left: 16px;
 }
 
+.BoardButtonDetails__fields .InputImage {
+  width: 150px;
+  height: 50px;
+}
+.BoardButtonDetails__fields .InputImage .InputImage__label {
```

Comment:
No need to over qualify selectors you can reach the same result with `.BoardButtonDetails__fields .InputImae__label`